requirements_all.txt

```diff
@@ -375,7 +375,7 @@ bond-api==0.1.11
 
 # homeassistant.components.amazon_polly
 # homeassistant.components.route53
-boto3==1.9.252
+boto3==1.17.25
```

Comment:
This version isn't strictly compatible with aiobotocore, since boto3 1.17.25 requires botocore>=1.20.25,<1.21.0, but aiobotocore==0.11.1 [requires: botocore>=1.13.14,<1.13.15. The latest aiobotocore is 1.2.2 which could be compatible with boto3 1.16.10:

```
botocore==1.19.52
  - aiobotocore==1.2.2 [requires: botocore>=1.19.52,<1.19.53]
  - boto3==1.16.10 [requires: botocore>=1.19.10,<1.20.0]
    - pycognito==0.1.5 [requires: boto3>=1.10.49]
      - hass-nabucasa==0.41.0 [requires: pycognito==0.1.5]
    - pyhiveapi==0.3.4.4 [requires: boto3>=1.16.10]
  - pyhiveapi==0.3.4.4 [requires: botocore>=1.19.10]
  - s3transfer==0.3.4 [requires: botocore>=1.12.36,<2.0a.0]
    - boto3==1.16.10 [requires: s3transfer>=0.3.0,<0.4.0]
      - pycognito==0.1.5 [requires: boto3>=1.10.49]
        - hass-nabucasa==0.41.0 [requires: pycognito==0.1.5]
      - pyhiveapi==0.3.4.4 [requires: boto3>=1.16.10]
```

Reply:
Looks like we can bump that as well. I looked though the changelog and the only breaking change that looked like an issue was the 1.0.0 version but it looks like we are already using it as an async context manager

Reply:
Unfortunately, the breaking change is actually an issue.

Reply:
I checked again, and boto3==1.16.52 is the highest version that still works with botocore 1.19.52.

Reply:
Its an undocumented breaking change that is the problem